src/lib/fetch.ts

```diff
@@ -993,6 +1017,76 @@ async function fetchActions(
   return actionsList;
 }
 
+/**
+ * Gets random key pair (public and private keys) from accounts manager
+ * that operates with accounts configured in network's Genesis Ledger.
+ *
+ * If an error is returned by the specified endpoint, an error is thrown. Otherwise,
+ * the data is returned.
+ *
+ * @param isRegularAccount Whether to acquire regular or zkApp account (one with already configured verification key)
+ * @param accountsManagerEndpoint Accounts manager endpoint to fetch from
+ * @returns Key pair
+ */
+async function acquireKeyPair(
+  isRegularAccount = true,
+  accountsManagerEndpoint = networkConfig.accountsManagerEndpoint
```

Comment:
to avoid having a unnamed boolean in the API, I think you should make this function take an object, `{ isRegularAccount, accountManagerEndpoint }`. object should be optional because both properties are